### ISSUE TITLE
Error while installing netlify with npm install netlify-cli -g 

### ISSUE THREAD

**Issue by msaharan** [netlify/cli#951]
I get the following error when I run the install command.
```
[dyskun@tux ~]$ npm install netlify-cli -g
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN checkPermissions Missing write access to /usr/lib/node_modules
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/netlify-cli/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/lib/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/lib/node_modules'
npm ERR!  [Error: EACCES: permission denied, access '/usr/lib/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/lib/node_modules'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dyskun/.npm/_logs/2020-06-23T03_57_12_339Z-debug.log

```
System information:
OS: Manjaro Linux
npm version: 6.14.5
node.js version: v14.4.0

A [similar issue](https://github.com/netlify/cli/issues/769#issuecomment-610827651) has been raised earlier.

**Comment by erquhart** [comment]
There's a page on solving this issue in the npm docs, I would recommend trying that first: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

**Comment by msaharan** [comment]
> There's a page on solving this issue in the npm docs, I would recommend trying that first: https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

That worked. Thanks. This is the first time I am working with node.js so I couldn't have found that page myself. 

For anyone else who might face this issue. Following is what I did. 

- [This page](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally) says that we do not need to remove the current version of npm and node.js but I still removed it. 
- [This page](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) recommends that we use a node version manager.
- I Installed the nvm.
```
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash

```
- Then I installed the LTS version of node.js.
```
npm install --lts
```
- Then the installation command worked.
